### ISSUE TITLE
Displaying Audio Waveforms for Multi-Track MP4 Files in AutoEditor UI Version

### ISSUE THREAD

**Issue by YDeltagon** [basswood-io/auto-editor-app-issues#3]
### Bug description

When importing an MP4 file with five audio tracks from OBS, AutoEditor's UI does not display any audio waveforms. 
I have to cut and restart the software multiple times to make each waveform appear one by one.

### What's your platform?

```shell
Windows 11, 64-bit, AMD Ryzen 7 5800X3D, Radeon RX 7900XTX, and 32GB of RAM
AutoEditor 2.19.0
OBS 30.0.0

48000Khz
16Bit audio
320KBPS
AAC

MP4 converted from MKV with 5 tracks
```

**Comment by YDeltagon** [comment]
IDK why but, Before, I used to record in .MKV with OBS, and I let it convert to .mp4 (advanced option in OBS). Now, I just tried recording directly in .MP4 (which might be problematic if my PC BSODs, but oh well) and it works, the two issues are resolved, so it wasn't coming from this app, but from OBS, sorry, I'm closing both :)

**Comment by YDeltagon** [comment]
Hello,

I'm experiencing a recurring issue where AutoEditor fails to display audio waveforms for multi-track MP4 files, even after multiple troubleshooting attempts. Here's a detailed description of the problem and the steps I've taken so far:

Environment:
OS: Windows 11 (fresh install)
CPU/GPU: AMD
AutoEditor Version: 2.22.2
Drivers: Updated to the latest versions

Issue Description:
The AutoEditor UI shows the video and play button but no audio waveforms appear in the timeline. This issue persists across various file formats and videos.

Troubleshooting Steps Attempted:
Rebooted the system several times.
Reinstalled AutoEditor.
Tried importing different video formats and files.
None of these steps have resolved the issue. Previously, I found that recording directly in MP4 format (as opposed to converting from MKV) seemed to solve the problem temporarily. However, this workaround is no longer effective.

Could someone please assist me in resolving this issue? Any suggestions or insights would be greatly appreciated!

**Comment by YDeltagon** [comment]
![image](https://github.com/basswood-io/auto-editor-app-issues/assets/70691658/abba007e-0d6d-4c08-acd3-3d2d65f5a289)

![image](https://github.com/basswood-io/auto-editor-app-issues/assets/70691658/36518e83-3e9d-4855-848f-22070d4f4e8d)


Some error : 
![image](https://github.com/basswood-io/auto-editor-app-issues/assets/70691658/0668bcb4-5e18-4c79-8c08-292ac304fbf4)

**Comment by YDeltagon** [comment]
![image](https://github.com/basswood-io/auto-editor-app-issues/assets/70691658/b51eee94-5405-43be-bbb0-3c168591fb08)
**Version 2.22.1 Work FINE !**

**Comment by YDeltagon** [comment]
![image](https://github.com/basswood-io/auto-editor-app-issues/assets/70691658/6ead6965-3cf1-45ff-a264-ebe5eee64937)
No error on 2.22.1

**Comment by WyattBlue** [comment]
This closed because I'm pretty sure this is fixed in the `3.0` and `3.0.1`.
